platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitBasicAuthenticationCredentialsProvider.java

```diff
@@ -0,0 +1,46 @@
+package org.eclipse.che.api.git;
+
+import org.eclipse.che.api.git.server.dto.DtoServerImpls;
+import org.eclipse.che.api.git.shared.GitUser;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by I053322 on 9/4/2016.
+ */
```

Comment:
Please make this javadock in common way:
`/**`
` * <description>`
` *`
` * @author <author>`
Like you did it in PR to 4.x